frontend/src/views/Account.vue

```diff
@@ -72,6 +83,11 @@ const availableLanguages = [
 	{ value: "hi", label: "हिंदी" },
 ]
 
+const appVersion = createResource({
+	url: "changemakers.api.get_app_version",
+})
+appVersion.fetch()
```

Comment:
```suggestion
const appVersion = createResource({
	url: "changemakers.api.get_app_version",
auto: true
})
```

should have the same effect